decoder/vaapidecoder_h264.cpp

```diff
@@ -1760,7 +1760,7 @@ YamiStatus VaapiDecoderH264::decodeSlice(NalUnit* nalu)
         status = createPicture(slice, nalu);
         if (status != YAMI_SUCCESS)
             return status;
-        if (!m_currPic
+        if (!m_currPic || !m_prevPic
```

Comment:
No, the first frame may not be an IDR.  Based on our testing, the first frame could be a SEI if the video contains SEI frames.  This is probably a mediaplayer's flaw, but it will be better if yami can handle this kind of defects.